spec/nesting/not.hrx

```diff
@@ -0,0 +1,9 @@
+<===> with-parent/input.scss
```

Comment:
```suggestion
<===> multiple_parent_selectors_with_trailing_ident/input.scss
```

It's good to be explicit about what's being tested, especially when it's a case like this that was triggered by a confluence of different factors.